packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,217 @@
+   */
+  isBorderless?: boolean;
+
+  /**
+   * Renders a primary action button and a secondary action link button
+   */
+  actions?: {
+    /**
+     * Renders a button
+     */
+    primary?: PrimaryAction;
+
+    /**
+     * Renders a Link button
+     */
+    secondary?: SecondaryAction;
+  };
+};
+
+// todo: add all icons
+const intentIconMap = {
+  positive: AlertIcon,
+  negative: AlertIcon,
+  information: AlertIcon,
+  notice: AlertIcon,
+};
+
+const Alert = ({
+  description,
+  title,
+  isDismissable = true,
+  onDismiss,
+  contrast = 'low',
+  isFullWidth = false,
+  intent = 'information',
+  isBorderless = false,
+  actions,
+}: AlertProps): ReactElement | null => {
+  const [isVisible, setIsVisible] = useState(true);
+  const contrastType = contrast === 'high' ? 'highContrast' : 'lowContrast';
+
+  const Icon = intentIconMap[intent];
+  const icon = <Icon color={`feedback.icon.${intent}.${contrastType}`} size="large" />;
+
+  const _title = title ? (
+    <Box marginBottom="spacing.1">
+      <Heading variant="small" contrast={contrast}>
+        {title}
+      </Heading>
+    </Box>
+  ) : null;
+
+  const _description = <Text contrast={contrast}>{description}</Text>;
+
+  const primaryAction = actions?.primary ? (
+    <Box marginRight="spacing.4" display="inline-flex">
+      <BaseButton onClick={actions.primary.onClick} intent={intent} contrast={contrast}>
+        {actions.primary.text}
+      </BaseButton>
+    </Box>
+  ) : null;
+  const secondaryActionParams: Nullable<Partial<SecondaryActionLinkButton>> = actions?.secondary
+    ? {
+        onClick: actions.secondary.onClick,
+      }
+    : null;
+  if (actions?.secondary && secondaryActionParams && 'href' in actions.secondary) {
+    // type guard with href to ensure this is now a SecondaryActionLinkButton
+    secondaryActionParams.href = actions.secondary.href;
+    secondaryActionParams.target = actions.secondary.target;
+    secondaryActionParams.rel = actions.secondary.rel;
+  }
+  const secondaryAction = actions?.secondary ? (
+    // Todo: Link font weight mismatch
+    <BaseLink contrast={contrast} intent={intent} {...secondaryActionParams}>
+      {actions.secondary.text}
+    </BaseLink>
+  ) : null;
+
+  const _actions =
+    primaryAction || secondaryAction ? (
+      <Box marginTop="spacing.3">
+        {primaryAction}
+        {secondaryAction}
+      </Box>
+    ) : null;
+
+  const onClickDismiss = (): void => {
+    if (onDismiss) {
+      onDismiss();
+    }
+    setIsVisible(false);
+  };
+  const closeButton = isDismissable ? (
+    <IconButton
+      accessibilityLabel="Dismiss alert"
+      onClick={onClickDismiss}
+      contrast={contrast}
+      size="large"
+      icon={CloseIcon}
+    />
+  ) : null;
+
+  const a11yProps = makeAccessible({
+    role: intent === 'negative' || intent === 'notice' ? 'alert' : 'status',
+    ...(intent === 'notice' && { liveRegion: 'polite' }), // override the implicit live region with role alert
```

Comment:
nit: 
```suggestion
    // override the implicit live region of role `alert`
    ...(intent === 'notice' && { liveRegion: 'polite' }),
```

Reply:
Done in https://github.com/razorpay/blade/pull/654/commits/ee1c1bbada067d0954583da2d27b9ffd898dee65